src/applications/find-va-forms/actions/index.js

```diff
@@ -0,0 +1,8 @@
+export const QUERY_CHANGED = 'QUERY_CHANGED';
+
+export function updateQuery(query) {
```

Comment:
I've been noticing there's a lot of function expressions over arrow functions in the codebase. How come we do this?

Reply:
In general, I think we only use the ES5 for top-level declarations, and only because the ES6 arrow functions are used everywhere else, so it looks a little clearer. I don't think that is an official pattern though, and since this is a new widget though, we can use the ES6 if you prefer. Would you like me to update that?

Reply:
Oh interesting! I personally prefer es6 arrow functions, otherwise it makes me think we're going to be using `this` differently, but I'll leave it up to you to decide! 👍

Reply:
Updated 🙂